docs/guided-remediation.md

```diff
@@ -243,11 +302,320 @@ osv-scanner fix --non-interactive --format json --strategy=relax -M path/to/pack
 
 </details>
 
+```bash
```

Comment:
should we mention that this format is expected to be stable and is intended for integration into automation (unlike text) ?